lib/Mojo/mysql/PubSub.pm

```diff
@@ -238,7 +312,7 @@ drop table if exists mojo_pubsub_subscribe;
 drop table if exists mojo_pubsub_notify;
 
 create table mojo_pubsub_subscribe (
-  id integer auto_increment primary key,
+  id bigint unsigned auto_increment primary key,
```

Comment:
You need to add a new `-- 2 down` and `-- 2 up` to make schema changes. (Same below)

Reply:
I think I'm just going to pull this change.  Without using DDL tools, there's no safe way to change the column def on a live database without locking the table.  Plus it was more an issue before I realized the `REPLACE` commands were chewing up primary keys, and the `INSERT ... ON DUPLICATE KEY UPDATE` gets rid of that problem.